frontend/src/view/UserComment/UserComment.js

```diff
@@ -0,0 +1,34 @@
+import { StyledCommentContainer } from './CommentContainer.styled';
+import { StyledLeftSection } from './LeftSection.styled.js';
+import { StyledRightSection } from './StyledRightSection';
+import { useState } from 'react';
+import { UserAvatar } from '../UserAvatar/UserAvatar';
+
+export const UserComment = ({ commentObject }) => {
```

Comment:
```suggestion
export const UserComment = ({ comment }) => {
```

Raczej z założenia wychodzimy, że "wszystko jest w JS obiektem". Raczej nie powinniśmy dodawać `object` do nazwy (są pewne wyjątki)